frontend/constants/serviceTemplates.ts

```diff
@@ -17,7 +17,7 @@ export const KPI_DESC_PREFIX = '[Pearl service]';
 export const PREDICT_SERVICE_TEMPLATE: ServiceTemplate = {
   agentType: AgentType.PredictTrader, // TODO: remove if causes errors on middleware
   name: 'Trader Agent', // should be unique across all services and not be updated
-  hash: 'bafybeihnwktiqopqvztnoohfs4bvdwvliaba32uzet2f6jkflycph3msie',
+  hash: 'bafybeihoutfbf7xp7oj5reskoubvlaasajrmuwxdptcytqnowusfmqtd7y',
```

Comment:
```suggestion
  hash: 'bafybeie2bn7yhrjqjx7j3lbyjd6ds2t4m5m22bfomwl36ajjthlfaxfauq',
```

New hash from https://github.com/valory-xyz/trader/pull/604/files#diff-e459cec4c3e84b7f458724acf6256afb5eb4d16d0fe4f9ff97db9a57e95e8cbb